apps/mosaico/src/Mosaico/Frontpage.purs

```diff
@@ -92,6 +92,10 @@ render (List props) =
                                       { href: "/artikel/" <> a.uuid
                                       , children: [ DOM.h2_ [ DOM.text $ fromMaybe a.title a.listTitle] ]
                                       }
+                                  , DOM.span
+                                    { className: "list-article-timestamp"
+                                    , children: [ DOM.text $ foldMap formatLocalDateTimeAsArticleTime a.publishingTime ]
```

Comment:
Nothing wrong with this, however if the only purpose of `formatLocalDateTimeAsArticleTime` is to format the underlying `DateTime` (and we only use it here), we could more simply unwrap it and format in place. As `LocalDateTime DateTime` is a [Newtype](https://pursuit.purescript.org/packages/purescript-newtype/5.0.0/docs/Data.Newtype), it's convenient also:

```
foldMap (formatArticleTime <<< unwrap) a.publishingTime
```

Note that this `DOM.span { className: "list-article-timestamp" ...` is always rendered even if we have no publishing time. This shouldn't ever happen though, as we aren't returning drafts in search results.